src/admin/class-wordlift-install-wizard.php

```diff
@@ -0,0 +1,759 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * The install wizard
+ *
+ * @link       http://wordlift.it
+ * @since      3.9.0
+ *
+ * @package    Wordlift
+ * @subpackage Wordlift/admin
+ */
+
+/**
+ * The install wizard.
+ *
+ * Methods to track and implement the various steps of the install wizard
+ * which is triggered on dirst install of the plugin (when there are no settings)
+ *
+ * @package    Wordlift
+ * @subpackage Wordlift/admin
+ * @author     WordLift <hello@wordlift.it>
+ */
+class Wordlift_Install_wizard {
+
+	/**
+	 * Initialize the class and set its properties.
+	 *
+	 * @since    3.9.0
+	 *
+	 */
+	public function __construct(  ) {
+
+		$option = get_option('wl_general_settings_wizard',false);
+		if (false === $option)
+			$this->step = 1;
+		else
+			$this->step = $option;
+		
+		add_action( 'wp_loaded', array( $this, 'hide_notices' ) );
+		add_action( 'admin_menu', array( $this, 'admin_menu' ) );
+		add_action( 'admin_init', array( $this, 'show_page' ) );
+		add_action( 'admin_notices', array( $this, 'admin_notices' ) );
+	}
+
+	/**
+	 * Generate an admin notice sugesting to start the wizard if there is no configuration
+	 *
+	 * @since    3.9.0
+	 *
+	 */
+	public function admin_notices() {
+
+		$show = true;
+		$o = get_option('wl_general_settings',false);
+		
+		// an indication that setting was already done is that there is more to the settings then the key
+		// when a user skipps the wizard another index is added to the array in addition to the key
+		if (is_array($o) && (count($o) > 1)) 
+			$show = false;
+			
+		if ($show) {
+		?>
+		<div id="wl-message" class="updated">
+			<p><?php _e( '<strong>Welcome to WordLift</strong> &#8211; You&lsquo;re almost ready to start', 'wordlift' ); ?></p>
```

Comment:
Change to `esc_html_e` (see [Use esc_html_e() instead of _e()](https://github.com/Automattic/_s/issues/231))